scripts/templates/index.d.ts.tpl

```diff
@@ -5,6 +5,7 @@
 /// <reference lib="esnext.asynciterable" />
 
 import * as http from 'http'
+import { OctokitRequestOptions } from "@octokit/request/dist-types/types";
```

Comment:
If we depend on internal APIs we have to pin the `@octokit/request` version in package.json